qualang_tools/results/results.py

```diff
@@ -36,6 +36,10 @@ def __init__(self, job, data_list, mode="wait_for_all"):
                     self.data_handles[-1].wait_for_values(1)
                 else:
                     raise Warning(f"{data} is not saved in the stream processing.")
+            # Live plotting parameters
+            self.__bcont = False
+            self.__blast = True

```

Comment:
why two underscores? Usually, one is used for internal parameters.

Reply:
I saw that it is two '_' for making variables private: https://www.tutorialspoint.com/private-variables-in-python

Reply:
Interesting, I didn't know that. But that page is a bit misleading:
https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name
A single _ is for making them "private" (not intended to be changed/accessed by the user). 
A double __ is for making them "private" (not to be changed by sub-children of the class).
We care about the first :)

Reply:
So what I wanted is that the user cannot access these variables which can only be done with double '_' otherwise you can still get them by typing results._blast

Reply:
The user can still get them with double `__`, he just need to type a bit more (`results._fetching_tools_start_time`).

In any case, in Python there is no real hidden/private variables, you mark them with a single `_`. If the user wants, he can get them, but we're ok with that.

Reply:
btw, blast is confusing as it is a word in english... I would change to `_b_last`

Reply:
Ok I don't mind